dashboard/app/views/pd/workshop_mailer/teacher_enrollment_reminder.html.haml

```diff
@@ -31,6 +31,11 @@
     = survey_link
     as soon as possible. This will help your facilitator plan the workshop.
 
+- if @workshop.course == Pd::Workshop::COURSE_CSP
+  = render partial: 'csp_supply_list'
+- elsif @workshop.course == Pd::Workshop::COURSE_CSD
+  = render partial: 'csd_supply_list'
```

Comment:
Rather than branching in the view, it might be nice resolve this in the controller and pass the partial name as a parameter, like we do [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/mailers/pd/workshop_mailer.rb#L8-L26). With only 2 options it's not a super big deal. Your call.